### ISSUE TITLE
Another page

### ISSUE THREAD

**Issue by MiguelLemos02** [rizzdev/bad_help#1]
when i try to add updates the page does not add(type another page), could you give an example of adding another page?

**Comment by SkinnyDragonCodes** [comment]
Agreed, the way this is written is very confusing. Especially someone who is new to JS/ HMTL....
